### PR TITLE
Run the link checker (and prepare for a scheduled build with it)

### PR DESCRIPTION
**Pull request by ennru**
References #265

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -96,7 +100,7 @@ jobs:
 
 stages:
   - name: test
-    if: NOT tag =~ ^v
+    if: (NOT tag =~ ^v) AND (type != cron)
```
Comment (ignasi35) [inline .travis.yml:103]: What's the purpose of disabling tests in cron builds? 

If a test fails it prevents the publishing, so the question is whether that's a valid failure or a flaky test... hmmm.

Comment (ennru) [reply .travis.yml:103]: I don't think it is useful to run these tests when there are no changes and it spares us some CI cycles for other repos.

**Comment by ignasi35** [comment]
this build has a dirty cache in travis. You can either rebase over main once #313 is merged, or close the PR and create a new one.

File: .travis.yml (hunk 1 above)
Comment (ignasi35) [reply .travis.yml:97]: OFC! This is not using snapshot artifacts from upstream. 👍🏼

File: .travis.yml [hunk 2]
```diff
@@ -6,19 +6,13 @@ services:
 git:
   depth: 500
 
-before_install:
-  # using jabba for custom jdk management
-  - if [ ! -f ~/.jabba/jabba.sh ]; then curl -L -v --retry 5 -o jabba-install.sh https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh && bash jabba-install.sh; fi
-  - . ~/.jabba/jabba.sh
-  - jabba install ${JDK:=adopt@~1.11-0}
-  - jabba use ${JDK:=adopt@~1.11-0}
```
Comment (ignasi35) [inline .travis.yml:14]: Where did this go? Do you suggest we rely on the JVM installed by default in the `language: scala` flavor of TravisCI VMs?

**Pull request closed without merging**